dns_managed_zone_with_dnssec/main.tf

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Google Cloud Documentation: https://cloud.google.com/dns/docs/dnssec-config#creating
+# [START dns_managed_zone_with_dnssec]
+resource "google_dns_managed_zone" "example" {
+  name        = "example-zone-name"
+  dns_name    = "myzone.example.com."
```

Comment:
In the logs, I am seeing ": Error 400: Please verify ownership of the 'myzone.example.com.' domain (or a parent) at http://www.google.com/webmasters/verification/ and try again, verifyManagedZoneDnsNameOwnership"

How have you worked around this in other samples? Can we just example.com?

Reply:
Hi @betsy-lichtenberg - in the past, this TF samples worked fine with `myzone.example.com.` but not now! Let me test with `example.com.`